src/Filter/Filter.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { MouseEventHandler, useState } from "react";
+import type { RadioChangeEvent } from "antd";
+import { Input, Modal, Button, Radio, Space } from "antd";
+import { DownOutlined } from "@ant-design/icons";
+import DateComp from "./DateComp";
+
+const { Search } = Input;
+const onSearch = (value: string) => console.log(value);
+
+const Filter = () => {
+  const [isModalVisible, setIsModalVisible] = useState(false);
+  const [value, setValue] = useState("Today");
+  const [buttonVal, setButtonVal] = useState("Today");
+
+  const showModal = () => {
+    setIsModalVisible(true);
+  };
+
+  const handleCancel = () => {
+    setValue(buttonVal);
+    setIsModalVisible(false);
+  };
+
+  //Radio
+  const onChange = (e: RadioChangeEvent) => {
+    console.log("radio checked", e.target.value);
+    setValue(e.target.value);
+  };
+
+  const handleOk = (e: any) => {
+    alert(value);
```

Comment:
what is value  ?

Reply:
Which one is selected in option button that value will be get

Reply:
use e.target.value as it is more understandable and less prone to error

Reply:
Now i removed e:any because value will get from state so only i removed.